source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -153,6 +154,15 @@
             }
         });
 
+        this.isResposibleCalificating = ko.computed(function () {
```

Comment:
Here you are overriding a previous defined _observable_, it is very dangerous because maybe the view already reference to the another instance.